mltsp/tests/data/test_features.csv

```diff
@@ -1,4 +1,4 @@
-meta1,meta2,meta3,std_err,freq1_harmonics_amplitude_0
```

Comment:
I thought it'd be best to use simple features for this kind of testing since running Lomb-Scargle instead of just about anything else is a lot of overhead; I just checked and in practice it would probably only take ~10 more seconds across all tests to include this feature, so if you think it's better to leave it I can do that. No strong preference.